### PR TITLE
ISSUE-316 Multiple fixes regarding contact search auto-complete

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by chibenwa** [comment]
Cc @tieugiang91

File: tmail-backend/jmap/extensions-es/src/main/java/com/linagora/tmail/james/jmap/EmailAddressContactMappingFactory.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                    .endObject()
+                    .startObject("id")
+                        .field("type", "text")
+                    .endObject()
+                    .startObject("address")
+                        .field("type", "text")
+                    .endObject()
+                .endObject()
+            /*
+                // TODO add the analisis / tokenizer parameter as parameters of IndexCreationFactory...
```
Comment (Arsnael) [inline tmail-backend/jmap/extensions-es/src/main/java/com/linagora/tmail/james/jmap/EmailAddressContactMappingFactory.java]: Is that supposed to follow an update on James submodule?

Comment (chibenwa) [reply tmail-backend/jmap/extensions-es/src/main/java/com/linagora/tmail/james/jmap/EmailAddressContactMappingFactory.java]: No it requires a ticket and a contribution in /backends-common/es-v7

Comment (Arsnael) [reply tmail-backend/jmap/extensions-es/src/main/java/com/linagora/tmail/james/jmap/EmailAddressContactMappingFactory.java]: yeah that's what I had in mind with my question

**Pull request merged**